### ISSUE TITLE
Amount not filling in with scanned QR code.

### ISSUE THREAD

**Issue by shortwavesurfer2009** [m2049r/xmrwallet#969]
I think the last update broke something, because now it says that I can exchange for other cryptos if I need to, but I tried to scan a QR code for a Haveno trade, and the amount is not filling itself in. Even though the payment request QR code has it.

It filled in the address and the transaction description properly. But when I click Next, it normally shows the amount already filled into the box and it is no longer doing that.

**Comment by anhdres** [comment]
Yep, it looked like we broke something when we added the sticky currency setting. Will fix asap.
